csrc/ir/nodes.cpp

```diff
@@ -464,6 +464,17 @@ std::vector<PolymorphicValue> UnaryOp::evaluate(
     case UnaryOpType::Sin:
       return {in.as<at::Tensor>().sin()};
       break;
+    case UnaryOpType::Signbit:
```

Comment:
Question for @Priya2698 .

Are we supposed to not have the full coverage on operations from expr eval ?

Reply:
We do intend on having full coverage.
When resolving Issue #670, we targeted developing for all operators seen in test cases. As we add more test cases, and missing operators are seen, we should implement evaluate methods for them.

Feel free to open and assign these issues to me, to address missing operators.

Reply:
I'm not trying to push this out as high priority. I think it's reasonably fine to patch it as issues surface.